cyvlfeat/sift/phow.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import matplotlib.colors as colors
+import dsift
+
+
+def phow(image, verbose=False, fast=True, sizes=[4, 6, 8, 10], step=2, color='gray',
+         floatdescriptors=False, magnif=6, window_size=1.5, contrastthreshold=0.005):
```

Comment:
Please don't maintain matlab CamelCase/single word and instead use the Python standard underscore separated. E.g `floatdescriptors` becomes `float_descriptors`.